src/components/text-input/TextInput.tsx

```diff
@@ -67,18 +81,21 @@ export const TextField = forwardRef<HTMLInputElement, PropsWithLabels<Props>>((p
     messageValid,
     prefix,
     suffix,
-    validation,
+    type = 'text',
+    validation = FormFieldValidation.None,
```

Comment:
[non-blocking] A default value doesn't need to be set here, the form field will set the default to `None`.